projects/zetasql/Dockerfile

```diff
@@ -26,7 +26,7 @@ RUN curl https://bazel.build/bazel-release.pub.gpg | apt-key add - && \
 
 RUN apt-get update && apt-get -y install bazel-1.0.0
 
-# RUN git clone --depth 1 <git_url> zetasql     # or use other version control
-# COPY zetasql-fuzzing zetasql
+RUN git clone -b simple_fuzzer --depth 1 \
```

Comment:
We'll want to remove the branch after the other PR goes in.

Reply:
Is it done ?